frontend/src/pages/SingleProcessPage.tsx

```diff
@@ -0,0 +1,84 @@
+import { Table } from "../components/Table";
+import { gql, useQuery } from "@apollo/client";
+import { useParams } from "react-router-dom";
+import { BpmnViewer } from "../components/BpmnViewer";
+import { styled } from "@mui/system";
+
+export default function ProcessesPage() {
+  const params = useParams();
+  const PROCESS = gql`
```

Comment:
Please choose consistent style on where to put the GraphQL query string. Here it is inside the function, whereas in the processes page it is outside it.